### PR TITLE
Loglevels are adjusted

### PR DESCRIPTION
**Pull request by annando**
A lot of log levels have been changed to a more appropiate level.

### CODE REVIEW

**Review by Quix0r — APPROVED** [review]
I noticed also a few other smaller changes in this changeset, e.g. COMPRESSED row format is being added.

**Review by MrPetovan — APPROVED** [review]

**Comment by annando** [comment]
I'm now happy with these changes. I had a look at the log entries of the last few hours and every created warning looks like something that I should have a deeper look at.

File: mod/tagger.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ function tagger_content(App $a) {
 
 	$item_id = ((DI::args()->getArgc() > 1) ? trim(DI::args()->getArgv()[1]) : 0);
 
-	Logger::notice('tagger: tag ' . $term . ' item ' . $item_id);
+	Logger::info('tagger: tag ' . $term . ' item ' . $item_id);
```
Comment (nupplaphil) [inline mod/tagger.php]:
```suggestion
	Logger::info('tagger: tag', ['term' =>  $term, 'item' => $item_id]);
```

File: src/Core/Worker.php [hunk 2]
```diff
@@ -104,7 +104,7 @@ public static function processQueue(bool $run_cron, Process $process)
 			foreach ($r as $entry) {
 				// The work will be done
 				if (!self::execute($entry)) {
-					Logger::notice('Process execution failed, quitting.');
+					Logger::warning('Process execution failed, quitting.');
```
Comment (nupplaphil) [inline src/Core/Worker.php]:
```suggestion
					Logger::warning('Process execution failed, quitting.', ['entry' => $entry]);
```

File: src/Model/Photo.php [hunk 3]
```diff
@@ -1095,7 +1095,7 @@ public static function upload(int $uid, array $files, string $album = '', string
 
 		$r = self::store($image, $user['uid'], 0, $resource_id, $filename, $album, 0, self::DEFAULT, $allow_cid, $allow_gid, $deny_cid, $deny_gid, $desc);
 		if (!$r) {
-			Logger::notice('Photo could not be stored');
+			Logger::warning('Photo could not be stored');
```
Comment (nupplaphil) [inline src/Model/Photo.php]: Can we store some more context to better identify the root cause? Like resource id, uid,...

File: src/Model/Photo.php [hunk 4]
```diff
@@ -1182,7 +1182,7 @@ public static function uploadAvatar(int $uid, array $files, string $url = ''): s
 
 		$r = self::store($image, $uid, 0, $resource_id, $filename, $album, 4, self::USER_AVATAR);
 		if (!$r) {
-			logger::notice('profile image upload with scale 4 (300) failed');
+			logger::warning('profile image upload with scale 4 (300) failed');
```
Comment (nupplaphil) [inline src/Model/Photo.php]: Same as above, is more context useful here?

**Review by nupplaphil — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Review by Quix0r — CHANGES_REQUESTED** [review]
@nupplaphil mentioned a few changes I also would like to see.

File: src/Model/Photo.php (hunk 4 above)
Comment (annando) [reply src/Model/Photo.php]: Done

File: src/Model/Photo.php (hunk 3 above)
Comment (annando) [reply src/Model/Photo.php]: Done

File: src/Core/Worker.php (hunk 2 above)
Comment (annando) [reply src/Core/Worker.php]: Done

File: mod/tagger.php (hunk 1 above)
Comment (annando) [reply mod/tagger.php]: Done

**Comment by annando** [comment]
All changes are applied.

**Review by Quix0r — COMMENTED** [review]
`['foo' => $foo]` seem to be the accepted way.

**Pull request merged**